apps/beeswax/src/beeswax/server/dbms_tests.py

```diff
@@ -37,118 +34,139 @@
 
 
 class TestGetQueryServerConfig():
-
   def setUp(self):
     cache.clear()
-
-
+  
   def test_get_default(self):
-
+    
     with patch('beeswax.conf.HIVE_SERVER_HOST.get') as HIVE_SERVER_HOST:
       with patch('beeswax.conf.HIVE_SERVER_PORT.get') as HIVE_SERVER_PORT:
         HIVE_SERVER_HOST.return_value = 'hive.gethue.com'
-        HIVE_SERVER_PORT.return_value=10002
-
+        HIVE_SERVER_PORT.return_value = 10002
+        
```

Comment:
Nit: Can we try and remove all extra lines containing these extra spaces?

Reply:
done